examples/2d/rotation.rs

```diff
@@ -1,7 +1,6 @@
 //! Demonstrates rotating entities in 2D using quaternions.
 
-use bevy::math::ops;
-use bevy::prelude::*;
+use bevy::{math::ops, prelude::*};
```

Comment:
Examples have just had minor formatting changes made to their `use` statements based on `cargo +nightly fmt`. They are exempt from the lint requiring `core` and `alloc` where possible.